stratum/hal/bin/bcm/standalone/README.md

```diff
@@ -40,14 +40,14 @@ Note the **ONLPv2**!
 x86-64-<vendor-name>-<box-name>-32x-r0
 ```
 
-## Pre-build Docker image
+## Pre-built Docker image
 
 Stratum for Broadcom switches can be run inside Docker on the switch itself.
 As part of CI, we publish Stratum with a pre-compiled binary and a set of default configuration files as a [Docker container](https://hub.docker.com/repository/docker/stratumproject/stratum-bcm).
 
- - `cd stratum/hal/bin/bcm/standalone`
+ - `cd stratum/hal/bin/bcm/standalone/docker`
  - `docker pull stratumproject/stratum-bcm:latest`
```

Comment:
Should we remove the `docker pull` command? The `docker run` command should pull the image if the Docker can't find the image.

Reply:
It's still needed to get updates. I moved it to the top and added a comment.